### DISCUSSION TITLE
Github Actions as Required Status Checks

### DISCUSSION THREAD

**Discussion by Novodes** [community/community#25610] [Actions]
<p>Hi,<br>
I would like to set my GitHub Actions as required status checks so that I would be able to have protected branches and prevent commits from being pushed to specific branches if they don’t pass the github actions checks.<br>
But when I go to the branch protection rules, the github actions don’t appear in the list of checks to choose from:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/d/8dbb5718f7ee173c5158417d38e8df63a4859318.png" data-download-href="/uploads/short-url/kdOHgMnjYGI7JEjtjSM4zDTQMru.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/35224049/181092190-1d75bccb-6ab2-4362-a1f7-b79d3bf3ab29.png" alt="image" data-base62-sha1="kdOHgMnjYGI7JEjtjSM4zDTQMru" width="690" height="380" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8dbb5718f7ee173c5158417d38e8df63a4859318_2_690x380.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8dbb5718f7ee173c5158417d38e8df63a4859318_2_1035x570.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/d/8dbb5718f7ee173c5158417d38e8df63a4859318.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/d/8dbb5718f7ee173c5158417d38e8df63a4859318_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1360×750 46 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>As you can see here I do have two github actions that have been running for several weeks on my repo:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/8/88db5e1eb77b33149bf89a8054aba079b001a062.png" data-download-href="/uploads/short-url/jwGVB8JN4gKxERihxRzbkK6EL3c.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/35224049/181092191-0e76e0c1-1b81-4405-b909-6ef614c049ac.png" alt="image" data-base62-sha1="jwGVB8JN4gKxERihxRzbkK6EL3c" width="690" height="367" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/88db5e1eb77b33149bf89a8054aba079b001a062_2_690x367.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/8/88db5e1eb77b33149bf89a8054aba079b001a062_2_1035x550.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/8/8/88db5e1eb77b33149bf89a8054aba079b001a062.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/8/88db5e1eb77b33149bf89a8054aba079b001a062_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1324×705 52.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>So am I missing something?</p>

**Comment by laughedelic** [comment, marked as answer]
<p>You need to search for the <em>job</em> names, not the workflow names.</p>

**Comment by Novodes** [comment]
<p>Yes. Thanks. I also posted the same question on stackoverflow and they gave me the same answer there: <a href="https://stackoverflow.com/q/69177488/4441211" class="inline-onebox" rel="noopener nofollow ugc">How to set Github Actions as Required Status Checks - Stack Overflow</a></p>
